src/Components/Sidebar/Sidebar.jsx

```diff
@@ -1,8 +1,18 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import './Sidebar.css';
 import { ICONS, ACTIVE_ICONS, ACTIVE_CLASS } from '../../constants';
+import { signOut } from 'firebase/auth';
+import { auth } from '../../config/firebase.config';
 const Sidebar = () => {
+
+  const navigate = useNavigate();
+  const handleLogout = async () => {
+    await signOut(auth);
```

Comment:
why are we not handling error case here?

Reply:
Done